keras_rs/src/layers/embedding/jax/test_utils.py

```diff
@@ -1,23 +1,25 @@
 """JAX-specific test utilities for embedding layers."""
 
 import typing
-from typing import Any, Mapping, TypeAlias, Union
+from typing import Any, Mapping, TypeAlias, Union, TypeVar
 
 import jax
-import keras
-import numpy as np
 from jax import numpy as jnp
+from jax_tpu_embedding.sparsecore.lib.nn import embedding
 from jax_tpu_embedding.sparsecore.lib.nn import embedding_spec
 from jax_tpu_embedding.sparsecore.lib.nn.embedding_spec import FeatureSpec
 from jax_tpu_embedding.sparsecore.lib.nn.embedding_spec import TableSpec
-
+import keras
```

Comment:
Can you move these imports back to where they were?

Reply:
Done.